src/utils/experimentUpdatesHandler.js

```diff
@@ -94,24 +81,12 @@ const uglyTemporalFixedProcessingConfig = (processingConfig, filterName) => {
   return fixedProcessingConfig;
 };
 
-const onGEM2SUpdate = (experimentId, update, dispatch) => {
+const onGEM2SUpdate = (update, dispatch, experimentId) => {
   const processingConfig = update?.item?.processingConfig;
   if (processingConfig) {
     let fixedProcessingConfig = uglyTemporalFixedProcessingConfig(processingConfig, 'classifier');
     fixedProcessingConfig = uglyTemporalFixedProcessingConfig(fixedProcessingConfig, 'mitochondrialContent');
-    dispatch(loadedProcessingConfig(fixedProcessingConfig));
-
-    // adding default config to every filter with auto option
-    Object.keys(fixedProcessingConfig).forEach((key) => {
-      const currentObject = fixedProcessingConfig[key];
-      const settingsKey = Object.keys(currentObject).find((current) => currentObject[current].auto);
-      if (settingsKey) {
-        dispatch(updateProcessingSettings(
-          key, { defaultFilterSettings: fixedProcessingConfig[key][settingsKey] },
-        ));
-        dispatch(saveProcessingSettings(experimentId, key));
-      }
-    });
+    dispatch(loadedProcessingConfig(experimentId, fixedProcessingConfig, true));
```

Comment:
What happened here? I see `loadProcessingConfig` is much smaller, how was the actual improvement made?

Reply:
what improvement?